collect_app/src/main/java/org/odk/collect/android/formmanagement/metadata/FormMetadataParser.kt

```diff
@@ -42,6 +42,7 @@ object FormMetadataParser {
         val autoDelete = submission?.getAttributeValue(null, "auto-delete")
         val autoSend = submission?.getAttributeValue(null, "auto-send")
         val geometryXPath = getFirstGeopointXPath(model, mainInstanceRoot, body)
+        val isEntityForm = model.getAttributeValue(null, "entities-version") != null
```

Comment:
@lognaturel this will be easy to change if so, but we don't expect non entity forms to have an `entities-version` attribute right? I'd originally thought we'd detect the entity node itself, but I think this works just as well.

Reply:
> I'd originally thought we'd detect the entity node itself

This would be a little bit more complex so I wanted to avoid it and @lognaturel said that this approach should work well. See the conversation on slack: https://getodk.slack.com/archives/C03QX56DD8C/p1729870733881349?thread_ts=1729864922.336559&cid=C03QX56DD8C